### DISCUSSION TITLE
Save Copy of Checklist

### DISCUSSION THREAD

**Discussion by harperaa** [solodit/feedback#45] [New Feature Requests]
It would be great to be able to save our own instance of a checklist, even a customized one (eventually).  This will allow us to use a checklist for each audit, and keep track of each audit and its progress.  Eventually, this could evolve into a progress bar for each checklist and even a place to capture notes and findings, and ultimately generate a report... but that is a big list... please start with allowing us to save our own checklist, based on the template(s) you provide.

**Comment by hans-cyfrin** [comment]
We recognize the benefits of using a custom checklist for each user, and we would say that was considered during development.
But we would rather encourage all researchers to contribute to the public shared checklist.

Regarding the other functionalities like saving sessions for various audits with progress, we will consider them in the future.
Thanks anyway for the suggestion.
